### PR TITLE
Data Release 2024-02-27

### PR DESCRIPTION
**Pull request by hareesh-ms**

### CODE REVIEW

**Review by pradh — APPROVED** [review]
Diffs generally LG from what I can tell, right?

File: internal/server/statvar/golden/search_statvar/fem.json [hunk 1]
```diff
@@ -85015,204 +85397,6 @@
           "dcid": "Percent_NeverOrOccasionally_ConsumptionOfVegetables_In_Count_Person_TertiaryEducation_Female"
         }
       ]
-    },
-    {
-      "dcid": "dc/g/Person_ActivityFrequency-NeverOrOccasionally_EducationalAttainment-UpperSecondaryEducationPostSecondaryNonTertiaryEducation_Gender-Female_HealthBehavior",
```
Comment (ajaits) [inline internal/server/statvar/golden/search_statvar/fem.json:85020]: This block of SVs seem deleted. Need to check if that's due to a limit on number of SVs chosen.

**Comment by pradh** [comment]
I'm going to merge this to prepare for release.

**Pull request merged**